src/components/data-management/FileUploadModal.jsx

```diff
@@ -38,12 +37,11 @@ const FileUploadModal = (props) => {
   }, [filesList]);
 
   // Handle on Drop
-  const onDrop = (acceptedFiles) => {
-    const newList = [];
+  const onDrop = async (acceptedFiles) => {
     let filesNotInFolder = false;
     const filteredFiles = acceptedFiles
       // Remove all hidden files
-      .filter((file) => !file.name.startsWith('.'))
+      .filter((file) => !file.name.startsWith('.') && !file.name.startsWith('__MACOSX'))
```

Comment:
Can you explain what is this check for starting with __MACOSX? I dont know

Reply:
Creating zip files on macOS creates this irritating hidden directory. I remember a user a while ago complaining about these files being selected for upload and having to manually remove them, so I thought I might as well add this to the filtering logic while I was here.

Reply:
Interesting, thanks!